pkg/common/cns-lib/vsphere/utils.go

```diff
@@ -99,6 +99,16 @@ func IsVimFaultNotFoundError(err error) bool {
 	return isNotFoundError
 }
 
+// IsCnsSnapshotCreatedFaultError checks if err is the CnsSnapshotCreatedFault fault returned by
+// CNS CreateSnapshots API
```

Comment:
Add comments to explain what CnsSnapshotCreatedFault means.

Reply:
Done